odxtools/odxdoccontext.py

```diff
@@ -8,4 +8,6 @@
 
 class OdxDocContext(NamedTuple):
     version: Version
-    doc_fragments: list["OdxDocFragment"]
+
+    # the doc_fragments are either tuple(category,) or tuple(category, diag_layer)
+    doc_fragments: tuple["OdxDocFragment"] | tuple["OdxDocFragment", "OdxDocFragment"]
```

Comment:
is it just me or comment and type hint don't match ?

Reply:
As I see it, the comment is a bit sloppy. it should probably be something like
```
    # the doc_fragments are either tuple(doc_frag(category),) or tuple(doc_frag(category), doc_frag(diag_layer))
```

Reply:
I pushed your comment